fetcher/src/job/job.types.ts

```diff
@@ -22,7 +22,7 @@ interface IReducer {
 interface IDefinition {
   url: string
   method: string
-  headers: IHeader[]
+  headers: IHeader
```

Comment:
@nick-bisonai Shouldn't we keep it as array since now we have a correct definition of the `IHeader`?

Reply:
`IHeader` is an object type that can contain multiple key-values, so it can contain other properties as well without being an array

Reply:
Right, my bad! Thank you!